molgenis-core-ui/src/main/javascript/modules/react-components/CheckboxWithLabel.js

```diff
@@ -0,0 +1,31 @@
+// CheckboxWithLabel.js
+import React from 'react';
+
+class CheckboxWithLabel extends React.Component {
```

Comment:
What is the motivation to use a different style of writing React components? If we want to switch for a different style we should refactor current components.